public/lib/helpers/timeline_helper.js

```diff
@@ -7,6 +7,16 @@ define(function (require) {
     function TimelineHelper() {
     }
 
+    TimelineHelper.prototype.createItemTemplate = function (itemDict) {
+      return '<div title="index: ' + itemDict.indexId + ', startField: ' + itemDict.startField +
```

Comment:
could you try using lodash template ? https://lodash.com/docs/3.10.1#template
maybe it will improve readability